components/layout/topbar/topbar.styled.tsx

```diff
@@ -40,3 +40,7 @@ export const StyledInputBase = styled(InputBase)`
     padding-right: calc(1em + 10px);
   }
 `;
+
+export const Space = styled.div`
```

Comment:
What does this does? I've seen the usage but cannot figure it out...

Reply:
This `div` added space in the nav-bar by growing the flex container, for moving all elements after it to the left corner.

In the documentation they put it on a text field to push everything. 
[ButtonAppBar](https://mui.com/material-ui/react-app-bar/#ButtonAppBar.tsx)